README.md

```diff
@@ -84,13 +123,17 @@ Noto is also part of my [portfolio](https://alialbaali.com) projects. It showcas
 
 ## Contributions
 
-Contributions are always welcome! If you would like to contribute, please feel free to file an issue or create a PR.
+<p>
+  Contributions are always welcome! If you would like to contribute, please feel free to file an issue or create a PR.
+</p>
 
 #### Special thanks to everyone who has helped and contributed in the development.
```

Comment:
If you don't mind, I'd appreciate you adding your name somewhere in the README, maybe here? Up to you! :)